doc/database-model.md

```diff
@@ -0,0 +1,61 @@
+
+# Database Entity Relationship Diagram
+
+```mermaid
+erDiagram
+    USER {
+        String _id PK
+        String name
+        String googleID
+        String address
+        ObjectId[] dogs
```

Comment:
what happend if i want to add a cat? Better use a field userId in other entity to relation it

Reply:
this issue is solved by using PET as general, or you want to create a separate PET table and connect the user with their userId?

Reply:
i prefer to use userId inside PET entity and delete "pets" field in users

Reply:
done